pandas/tests/io/formats/test_to_markdown.py

```diff
@@ -0,0 +1,12 @@
+import pandas.util._test_decorators as td
+
+import pandas as pd
+
+
+@td.skip_if_no_tabulate
```

Comment:
this can probably use a pytestmark at the module level

Reply:
Does that mean to make a class and then apply the decorator to the whole class? If so, that's what I've done

Reply:
It'd be `pytestmark = pytest.importorskip("tabulate")` But this is fine too.